.travis.yml

```diff
@@ -34,10 +34,10 @@ jobs:
       env: PKGS="build build_config build_daemon build_modules build_resolvers build_runner build_runner_core build_test build_vm_compilers build_web_compilers example scratch_space"
       script: ./tool/travis.sh dartfmt dartanalyzer_0
     - stage: analyze_and_format
-      name: "SDK: 2.9.0; PKGS: build, build_config, build_daemon, build_modules, build_resolvers, build_runner_core, build_test, build_vm_compilers, scratch_space; TASKS: `dartanalyzer --fatal-warnings .`"
+      name: "SDK: 2.9.0; PKGS: build, build_config, build_daemon, build_resolvers, build_runner_core, build_test, build_vm_compilers, scratch_space; TASKS: `dartanalyzer --fatal-warnings .`"
       dart: "2.9.0"
       os: linux
-      env: PKGS="build build_config build_daemon build_modules build_resolvers build_runner_core build_test build_vm_compilers scratch_space"
+      env: PKGS="build build_config build_daemon build_resolvers build_runner_core build_test build_vm_compilers scratch_space"
```

Comment:
Should this file be reverted?

Reply:
We do need to remove these, build_modules is no longer compatible with the stable sdk.